pkg/utils/oci/oci_suite_test.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package oci
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/distribution/distribution/v3/configuration"
+	"github.com/distribution/distribution/v3/registry"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	helmRegistry "helm.sh/helm/v3/pkg/registry"
```

Comment:
typically, we don't use camelCase in import aliases 👀 

```suggestion
	helmregistry "helm.sh/helm/v3/pkg/registry"
```